datahub-web-react/src/app/entity/shared/containers/profile/EntityProfile.tsx

```diff
@@ -295,6 +284,10 @@ export const EntityProfile = <T, U>({
     const isLineageEnabled = entityRegistry.getLineageEntityTypes().includes(entityType);
     const showBrowseBar = (isBrowsable || isLineageEnabled) && !hideBrowseBar;
 
+    if (error) {
+        console.log(`Received an error! ${error}`);
+    }
+
```

Comment:
Is this a remnant of testing? Ideally we shouldn't push errors to the console. If it's for internal capture - let's send to a reporting agent, if it's user feedback let's show it in an error message in the UI.

Reply:
ah no this just got copied over since it's in acryl-main, didn't mean to add it. can remove since we already show the error to the user below it looks like